docs/products/opensearch/howto/import-opensearch-data-using-elasticsearch-dump.rst

```diff
@@ -0,0 +1,80 @@
+
+Dump OpenSearch index using ``elasticsearch-dump``
+==================================================
+
+It is a good practice to perform backups of your OpenSearch data to another storage service. This way you can access your data and restore it if something unexpected happens to it. 
+
+In this article, you can find out how to dump your OpenSearch data to an:
+
+* :ref:`OpenSearch cluster <copy-data-from-os-to-os>`
+* :ref:`AWS S3 bucket <copy-data-from-os-to-s3>`
+
+We will be using ``elasticsearch-dump`` `tool <elashttps://github.com/elasticsearch-dump/elasticsearch-dump>`__ for these processes. To install the ``elasticsearch-dump``, read the `instructions <https://github.com/elasticsearch-dump/elasticsearch-dump/blob/master/README.md>`_ on GitHub.
+
+.. _copy-data-from-os-to-os:
+
+Copying data from OpenSearch to Aiven OpenSearch
+------------------------------------------------
+
+Prerequisites
+~~~~~~~~~~~~~
+
+* Aiven for OpenSearch cluster 
+* OpenSearch cluster 
+* ``elasticsearch-dump`` `tool <elashttps://github.com/elasticsearch-dump/elasticsearch-dump>`__ installed
+
+To import the data we will use ``elasticdump`` command. This command works by copying the ``input`` index data to an specific ``output``. The ``input`` and ``ouput`` can be either an OpenSearch URL or a file path (local or remote file storage). In this particular case, we are using both URLs, one from an **OpenSearch cluster** and the other one from **Aiven for OpenSearch cluster**.
```

Comment:
Why don't we put the first sentence when we introduce ``elasticsearch-dump`` since it's common (and not specific to the OS -> OS migration)

Reply:
ok, I added to the upper level cause it is relevant for both.